python/src/opendp/extras/polars/__init__.py

```diff
@@ -866,10 +935,18 @@ def _make(scale, threshold=None):
                 global_scale=scale,
                 threshold=threshold,
             )
+        
+        # when the query has sensitivity zero or is behind an invariant
+        try:
+            m_zero = _make(0., threshold=None)
+            if m_zero.check(d_in, d_out):
+                # if the zero scale measurement is already private, return it
+                return m_zero
+        except OpenDPException:
+            pass
```

Comment:
Can both lines throw the error, or can the scope of the `try` be smaller? I don't know the big picture, but this worries me a little bit.